openpype/hosts/flame/api/plugin.py

```diff
@@ -644,3 +652,293 @@ def _create_parents(self):
 
 # Publishing plugin functions
 # Loader plugin functions
+
+class ClipLoader(avalon.Loader):
+    """A basic clip loader for Flame
+
+    This will implement the basic behavior for a loader to inherit from that
+    will containerize the reference and will implement the `remove` and
+    `update` logic.
+
+    """
+
+    options = [
+        qargparse.Boolean(
+            "handles",
+            label="Set handles",
+            default=0,
+            help="Also set handles to clip as In/Out marks"
+        )
+    ]
+
+    def load(
```

Comment:
Not sure if this is final stage but why are `load`, `update` and `remove` overriden if are not implemented?